front/components/assistant/conversation/ConversationTitle.tsx

```diff
@@ -85,83 +84,83 @@ export function ConversationTitle({
                 visual={user.pictureUrl}
                 size="md"
                 key={i}
-                isRounded
               />
             ))}
           </Avatar.Stack>
         </div>
 
-        <div className="hidden lg:flex">
-          {onDelete && (
-            <DropdownMenu>
-              <DropdownMenu.Button>
-                <Button
-                  size="sm"
-                  labelVisible={false}
-                  tooltipPosition="below"
-                  variant="secondaryWarning"
-                  label="Delete Conversation"
-                  icon={TrashIcon}
-                />
-              </DropdownMenu.Button>
-              <DropdownMenu.Items width={280}>
-                <div className="flex flex-col gap-y-4 px-4 py-4">
-                  <div className="flex flex-col gap-y-2">
-                    <div className="grow text-sm font-medium text-element-800">
-                      Are you sure you want to delete?
-                    </div>
+        <Button.List>
+          <div className="hidden lg:flex">
```

Comment:
Button.List does make assumptions if I recall correctly on the fact that the children are Button no?

Reply:
It does not — It does look if the Child is a Button in order to remove a magnifying effect if it is.
But if it's not, it's not an issue.

Reply:
kk